src/DaPigGuy/PiggyFactions/commands/FactionCommand.php

```diff
@@ -100,9 +100,13 @@ protected function prepare(): void
         if ($this->plugin->isFactionBankEnabled()) $this->registerSubCommand(new DepositSubCommand($this->plugin, "deposit", "Deposit money into faction bank"));
         $this->registerSubCommand(new DescriptionSubCommand($this->plugin, "description", "Set faction description", ["desc"]));
         $this->registerSubCommand(new DisbandSubCommand($this->plugin, "disband", "Disband your faction"));
-        $this->registerSubCommand(new EnemySubCommand($this->plugin, "enemy", "Mark faction as an enemy"));
+        if ($this->plugin->getConfig()->getNested("enemy", false)) {
+            $this->registerSubCommand(new EnemySubCommand($this->plugin, "enemy", "Mark faction as an enemy"));
+        }
         $this->registerSubCommand(new FlagSubCommand($this->plugin, "flag", "Manage faction flags"));
-        $this->registerSubCommand(new FlySubCommand($this->plugin, "fly", "Fly within faction territories"));
+        if ($this->plugin->getConfig()->getNested("fly", false)) {
+            $this->registerSubCommand(new FlySubCommand($this->plugin, "fly", "Fly dans son territoire"));
```

Comment:
Revert this message back to english.
```suggestion
            $this->registerSubCommand(new FlySubCommand($this->plugin, "fly", "Fly within faction territories"));
```